packages/askui-nodejs/src/execution/ui-control-client.ts

```diff
@@ -344,6 +348,38 @@ export class UiControlClient extends ApiCommands {
     }
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  private evaluateRelation(
+    command: FluentFiltersOrRelations,
+    relation: RelationsForConvenienceMethods,
+    text: string,
+  ) {
+    let commando = command;
+    switch (relation) {
+      case 'nearestTo':
+        commando = command.nearestTo().text(text);
+        break;
+      case 'leftOf':
+        commando = command.leftOf().text(text);
+        break;
+      case 'above':
+        commando = command.above().text(text);
+        break;
+      case 'rightOf':
+        commando = command.rightOf().text(text);
+        break;
+      case 'below':
+        commando = command.below().text(text);
+        break;
+      case 'contains':
+        commando = command.contains().text(text);
+        break;
+      default:
+        throw new ValidationError('No valid Relation.Type was passed.');
+    }
+    return commando;
+  }
```

Comment:
Just some minor ideas for improving the expressiveness and conciseness of the code (optional):

```suggestion
  private extendCommandWithRelationToText(
    command: FluentFiltersOrRelations,
    relation: RelationsForConvenienceMethods,
    text: string,
  ) {
    switch (relation) {
      case 'nearestTo':
        return command.nearestTo().text(text);
      case 'leftOf':
        return command.leftOf().text(text);
      case 'above':
        return command.above().text(text);
      case 'rightOf':
        return command.rightOf().text(text);
      case 'below':
        return command.below().text(text);
      case 'contains':
        return command.contains().text(text);
      default:
        throw new ValidationError(`Relation "{relation}" unknown. Please, use one of the following relations: {relationsForConvenienceMethods.map(r => `"{r}"`).join(", ")}`).;
    }
  }
```